domains/polishboiyt.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "nSwishBall",
+           "email": "",
+           "discord": "994187133847875604",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.XIuo3TEMd3zO072x-nuR77-FZoqubwnwodoBPz4UfgKEXB01jEWVaCGgys9MBvPTlNwoKIA2trv5oG20RgMZXTJMC1n0XRPsE2l2SMLMKdU5XCuZSCeWkYbNFo_lLH0G00Xu-bZyW9vyLk5-_fTUNxVc8w78mFIOSVh2LFVuz6MSCLPu5m95sZx92lWN_VQG9zuYWuixZW7IcjJJO6HMVhkeX_OIq9a4dsQYQqkwgqne7xtss8VKAiDs1lFGz6ln_se8VMvz9ueCM2oFJGoPY4WAKavhBpLgZhtL0UCpjhIr_NcPNm4Pj3_u2_LW2aX90dAkGTar3Yzt1TYGFleVNQ.U9h08vlrCAgPDnvRTBGXaA.Xi--eqysm5B-106c2SXHfzDa9m0IN9bK7_Xailz0FeLA-VymV8HOe9LeHqveUK_Dm9Mk8K11gez1aJumclh3duvPejink7aEU-cP-HU-Rnk.F4msy1IaCi1aO-IEH6Jr4Q"
+        },
+    
+        "record": {
+            "CNAME": "nswishball.github.io/new-page"
```

Comment:
Invalid CNAME, must be a hostname with no file paths.

Reply:
wait did i have to insert https:// before the nswishball link thingy?

Reply:
> Invalid CNAME, must be a hostname with no file paths.

I made a new page and it's not a filepath, it's another repository.
I forgot to put the "https://" at the beggining.